caravel/bin/caravel

```diff
@@ -17,6 +17,8 @@ from caravel import data, utils
 from caravel import db
 
 config = app.config
+logging.config.dictConfig(config.get('LOGGING_CONFIG'))
```

Comment:
is this line needed here and everywhere else? 
It's already called in `caravel/__init__.py` and that is executed whenever any caravel module is imported.

@sid88in